features/admin/suzie_interacts_with_the_dashboard.feature

```diff
@@ -1,20 +1,20 @@
-Feature: Admin views and edits buildings on dashboard
-  Scenario: Admin can see relevant content
+Feature: Suzie the admin views and edits buildings on dashboard
```

Comment:
I was under the impression that we use named personas for feature tests but do let me know if I went overboard and I can undo the commit :smile:

Reply:
AFAIK it's not compulsory but on the other hand, it is nicer and more BDD. I will update my new tests to match 👍